activemq-artemis/templates/svc.yaml

```diff
@@ -53,3 +57,26 @@ spec:
   type: LoadBalancer
   selector:
     app: {{ template "fullname" . }}
+
+{{ if .Values.domainName }}
```

Comment:
I think we should move this deployment to the installation of K8s. So we don't need to put it in a chart. We just allow dozer users to specify the DNS in the charts and that's all.